Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -246,9 +266,13 @@ class NewsTableViewController: UITableViewController,
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        let longPressRecognizer = UILongPressGestureRecognizer(target: self, action: #selector(self.handleLongPress))
-        longPressRecognizer.delaysTouchesBegan = true
-        tableView.addGestureRecognizer(longPressRecognizer)
+        if traitCollection.forceTouchCapability == .available {
+            registerForPreviewing(with: self, sourceView: tableView)
+        } else {
+            let longPressRecognizer = UILongPressGestureRecognizer(target: self, action: #selector(self.handleLongPress))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/ba933f4049a44fb3a8fc655051f4e975dcf6eb8e/Headlines/src/Controllers/NewsTableViewController.swift#L272"></span>Line should be 120 characters or less: currently 121 characters</td>
    </tr>
  </tbody>
</table>